src/services/service.ts

```diff
@@ -118,7 +118,7 @@ export abstract class Service<ApiResponse = any> {
   }
 
   public fetch<T = ApiResponse>(httpOptions: AxiosRequestConfig, options?: Options): Promise<AxiosResponse<T>> {
-    const headers: AxiosRequestHeaders = {
+    const headers: Record<string, string> = {
```

Comment:
Just out of interest what was the reason for this change?

Reply:
I'd updated the `axios` package which came with updated types and TS was no longer happy with this typing.
Best guess was that this previously was an alias for `Record<string, string>` and they've since updated it to be more strict